pycontw/pycontw-frontend
# feat(staff page)- update list

## Types of changes
* **Photo Update**

## Description

update new photo based on the list and drive

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Type “<localhost>/2023/en-us/about/staff” in url
2. Scroll and check if the new list is correct
3. Check again in “<localhost>/2023/zh-hant/about/staff”

## Expected behavior

<img width="1377" alt="image" src="https://github.com/pycontw/pycontw-frontend/assets/26858727/6b03f65a-1f2b-42b3-ad01-9c19fdc31f11">
<img width="1419" alt="image" src="https://github.com/pycontw/pycontw-frontend/assets/26858727/3b9f1345-17ca-4c49-9fe2-1efff5f58902">


## Related Issue

#438 
## Additional context
